data/json/npcs/hints.json

```diff
@@ -10,23 +10,24 @@
+    "Zombie brutes and hulks have really tough hide, but skeletal juggernauts with their bone plates are the worst.  Don't bother shooting at them with lower-caliber guns, the bullet will bounce right off!",
     "Try not to kill a boomer if it's standing right next to you.  They tend to explode when they die, and that pink goop will get all over you.",
-    "Skeletons are a tough target.  They're so skinny and full of holes that bullets fired at them will usually just fly right past.",
-    "Skeletons are too delicate to smash through doors or windows.  They also can't smell you, unlike zombies, so if you turn your light off at night you can sneak right past.",
+    "Skeletons are a tough target for a gun.  They're so skinny and full of holes that it's hard to make a good hit.  And those big ones are hard as nails too.",
+    "Small skeletons are too delicate to smash through doors or windows.  Big ones can walk in through a wall.  At least they can't smell you, unlike zombies, so if you turn your light off at night you can sneak right past.",
     "Don't try to take on a skeleton with a bladed weapon... you'll just leave scratch marks.  You've got to shatter those bones with a hammer or something.",
-    "It's a good idea to butcher corpses if you have the time, even if you don't plan on eating them.  I've seen these weird zombies bring their friends back from the dead!",
+    "It's a good idea to butcher corpses if you have the time.  I've seen these weird zombies bring their friends back from the dead!",
     "I have a buddy who was sleeping in this cabin way out in the woods, when he suddenly woke up to trees and vines growing right up through the floor and walls!  He said it was some kind of huge tree beast...",
     "Oh man, have you gone down into the old subway systems?  I'd be careful... there's these things down there that are like zombies, but tougher.",
     "There's snakes down in most of the old sewer systems.  They're slow on land, but boy, those suckers can swim fast!",
     "If you're planning on wandering around the sewers--not that you'd have a reason too--watch out for those fish.  Those suckers are fast, and vicious too!",
     "Have you seen those eyebots flying around?  It's hard to say, but some faction's controlling them--maybe the military.  All I know is, I don't want them taking my picture...",
     "Ever go toe-to-toe with a manhack?  Tiny little helicopters covered in knives.  Best be packing a shotgun!",
     "They're rare, but molebots are nasty things.  They bore through the ground, then pop up to stab ya.  Still, you're safe from them if you stay on pavement...",
-    "Don't fire your gun if you can help it--the noise attracts monsters!",
+    "Don't fire your gun if you can help it - the noise attracts monsters. If you could get a silencer, or make one, it would give you some advantage.",
```

Comment:
```suggestion
    "Don't fire your gun if you can help it - the noise attracts monsters.  If you could get a silencer, or make one, it would give you some advantage.",
```